src/chapters/data-and-variables/operators.rst

```diff
@@ -139,15 +143,12 @@ result DOWN to the next smallest integer, regardless of any decimal values.
    rounding. ``int(-99 / 10)`` returns ``-9``, but ``-99 // 10`` returns
    ``-10``.
 
-.. _modulus:
+.. _modulo:
```

Comment:
i changed the later reference to this

Reply:
Did you catch all of them?  I honestly cannot remember how many times in the other chapters and appendices we link back to this section, but there are a few.  Once we merge in the chapter 4 edits, any links we miss in chp 5+ will be broken.

Given a choice between trying to find and change all of the `modulus` links, it seems safer to just stick with the original, especially since the students won't see the reference text.  Yes, `modulo` is the proper term, but the links are active, and `modulus` serves the purpose.

Reply:
there was only later reference and I changed it so we're good

Reply:
OK.  I found another in the unmerged Assignment 2 branch, which I can fix after we merge in these edits.

Fingers crossed that these are all of them!